src/api/search/src/search.js

```diff
@@ -13,6 +13,8 @@ const createFieldsFromFilter = (filter) => {
   switch (filter) {
     case 'author':
       return ['author'];
+    case 'date':
+      return ['published'];
```

Comment:
The date seems like something you'd want to stack with other filters.  For example, I might want to search for posts by an author in a given date range.  I'm not sure that returning just `[published]` alone is all we want here.